src/core/inline-idl-parser.js

```diff
@@ -7,16 +7,18 @@ import { html } from "./import-maps.js";
 const idlPrimitiveRegex = /^[a-z]+(\s+[a-z]+)+\??$/; // {{unrestricted double?}} {{ double }}
 const exceptionRegex = /\B"([^"]*)"\B/; // {{ "SomeException" }}
 const methodRegex = /(\w+)\((.*)\)$/;
-const slotRegex = /^\[\[(\w+)\]\]$/;
+
+export const slotRegex = /\[\[(\w+(?: +\w+)*)\]\](\([^)]*\)(\.[^)]*\))?)*$/;
 // matches: `value` or `[[value]]`
 // NOTE: [[value]] is actually a slot, but database has this as type="attribute"
-const attributeRegex = /^((?:\[\[)?(?:\w+)(?:\]\])?)$/;
+const attributeRegex = /^((?:\[\[)?(?:\w+(?: +\w+)*)(?:\]\])?)$/;
 const baseRegex = /^(?:\w+)\??$/;
 const enumRegex = /^(\w+)\["([\w- ]*)"\]$/;
 // TODO: const splitRegex = /(?<=\]\]|\b)\./
 // https://github.com/w3c/respec/pull/1848/files#r225087385
 const methodSplitRegex = /\.?(\w+\(.*\)$)/;
-
+const slotSplitRegex = /\.?\/(.+)/;
```

Comment:
I'm not following this one. Can you write some examples?

Reply:
Just `{{Foo/[[bar]](what, ever)}},  gets split into "Foo/" and `[[bar]](what, ever)` below.

Reply:
to be clear: 
```
nonMethodPart = "Foo"
methodPart = "[[bar]](what, ever)"
```

Reply:
I'm confused about the `\.?` part..

Reply:
Why not `split("/")`?

Reply:
Just to be clear, we're talking about `slotSplitRegex`

Reply:
Oh, you are right... that should have been just /\/(.+)/

Reply:
> Why not split("/")?

Wanted to only split at first... although having other "/"s would be weird, but I'd rather keeps the parts separate. It makes the whole thing more sane when it outputs and errors.